src/axom/spin/tests/spin_bvh.cpp

```diff
@@ -318,18 +309,19 @@ void generate_aabbs_and_centroids2d(const mint::Mesh* mesh,
         xsum += node[mint::X_COORDINATE];
         ysum += node[mint::Y_COORDINATE];
 
-        xrange.include(node[mint::X_COORDINATE]);
-        yrange.include(node[mint::Y_COORDINATE]);
+        range.addPoint(
+          primal::Point<FloatType, 2>::make_point(node[mint::X_COORDINATE],
```

Comment:
Minor: Would the following work?
```cpp
        range.addPoint( primal::Point<FloatType, 2>{ node[mint::X_COORDINATE], 
                                                     node[mint::Y_COORDINATE] });
```

Reply:
The issue is that `node` is an array of doubles, but `FloatType` can be instantiated as either a float or a double. So the initializer list overload doesn't match.

Reply:
That makes sense -- thanks.

Alternatively, I suppose you can static cast, which might make that distinction a bit more explicit.

Reply:
I'll go with initializer lists + static_cast; it's probably best to be explicit wrt the floating-point conversion.